raindrops/test_raindrops.py

```diff
@@ -0,0 +1,45 @@
+from raindrops import raindrops
```

Comment:
Great work on the test cases; the only other valuable one is a number with factors 3, 5, and 7!